### ISSUE TITLE
please add a demo to the readme

### ISSUE THREAD

**Issue by bionicles** [quickwit-oss/tantivy#993]
**Is your feature request related to a problem? Please describe.**
To evaluate this, it would be helpful to have a way to try it out!

**Describe the solution you'd like**
Perhaps a docker container with a prebuilt index and you just launch it and navigate to a localhost website to search some list of documents?

Then just put a code block at the top of the readme, ```docker run ...``` and put a link to the localhost port where it would be available (perhaps this could be an argument?)

**Comment by fmassot** [comment]
Hi @bionicles, I think the tantivy spirit is to keep it as a library and put stuff like what you suggest in another repository. @fulmicoton did a really good job to show how to index & search [tantivy-cli project](https://github.com/tantivy-search/tantivy-cli), did you have a look at it ?

**Comment by bionicles** [comment]
nope, that could be a good option. Just mean to say, library or not, usage examples are clutch for folks evaluating multiple options
